### DISCUSSION TITLE
DLL as sidecar

### DISCUSSION THREAD

**Discussion by Xtr126** [tauri-apps/tauri#11687] [Q&A] [answered]
Is there a workaround for bundling a DLL as a sidecar? Currently it works only for files with .exe extension

Bundle solution doesn't work when you want to support both 64-bit and 32-bit

Or should I put 32 bit DLLs in a separate dir and deal with it by using a separate config file passed by --config 32bitconfig.json ( or does tauri have a separate config file for 32 bit like it has tauri.conf.windows.json)

Or would be the only clean solution to write a build.rs script in rust to implement renaming DLLs to bundle by ourselves?
Thanks!

My usecase is bundling mke2fs.exe from cygwin, it won't run without 2 DLLs from cygwin. Other mke2fs.exe binaries don't require DLLs but have runtime errors.

**Comment by FabianLars** [comment, marked as answer]
I'd use the resources feature with the config flag until https://github.com/tauri-apps/tauri/issues/8501 is implemented

**Comment by Xtr126** [comment]
Thanks, I searched but could not find that issue.
